src/services/mqtt_client/mqtt_listener.py

```diff
@@ -52,6 +54,9 @@ def start(self, config: MqttSetting, subscribe_topics: List[str] = None, callbac
             # Resubscribe logic is not necessary here, these topics are for this app and will clear out when we start
             topic: str = self.__make_topic((self.get_listener_topic_prefix(), '#'))
             subscribe_topics.append(topic)
+        if self.config.cloud:
+            topic: str = self.__make_topic((self.get_schedule_value_topic_prefix(), '#'))
+            subscribe_topics.append(topic)
```

Comment:
As we want to clear the unlisted schedules from local brokers as well, include this subscription topic for all. Move this on top and add without condition.